### DISCUSSION TITLE
audio load issue

### DISCUSSION THREAD

**Discussion by taher267** [katspaugh/wavesurfer.js#3399] [Q&A]
## Bug description
html audio can load this audio (https://www.listennotes.com/e/p/a5ae21acf75a43538b635cf6b089f0b3). it is actually redirect a mp3 file

## Environment
 - Browser: Chrome
 - any browser

## Minimal code snippet
// A basic example

import WaveSurfer from 'https://unpkg.com/wavesurfer.js@7/dist/wavesurfer.esm.js'

const wavesurfer = WaveSurfer.create({
  container: document.body,
  waveColor: 'rgb(200, 0, 200)',
  progressColor: 'rgb(100, 0, 100)',
  url: 'https://www.listennotes.com/e/p/a5ae21acf75a43538b635cf6b089f0b3',
});

wavesurfer.on('click', () => {
  wavesurfer.play()
})

## Expected result
load and play 

## Obtained result
cors errors due to redirect another url

## Screenshots
https://prnt.sc/Y5O4E0ksfDXq

**Comment by katspaugh** [comment]
This is not a bug. It’s the expected behavior of fetch. Your redirect server should have CORS enabled as well as the destination server.
